lighthouse-core/audits/byte-efficiency/unminified-javascript.js

```diff
@@ -61,7 +62,9 @@ class UnminifiedJavaScript extends ByteEfficiencyAudit {
     const wastedBytes = Math.round(totalBytes * wastedRatio);
 
     return {
-      url: networkRecord.url,
+      url: networkRecord && !inline ?
```

Comment:
super nit: it feels like the inline is the special case we need to check for so it reads a bit easier in my head to say

an inline thing is something that's explicitly marked as such or doesn't have a network record
inline things get a prefix and content while everything else has a URL like normal